WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -805,6 +807,7 @@ class AbstractPostListViewController: UIViewController,
 
         refreshAndReload()
 
+        showRefreshingIndicator()
```

Comment:
We should show this indicator for the Published tab after opening Posts/Pages initially as well. Now Published tab only shows the indicator after switching filters. What do you think?

https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/27dddc6c-d94d-4456-a1d7-f0e6ab5c03c5

Reply:
Good catch. I should've re-rested it after I initially wrote it. I also noticed that the initial refresh starts in `viewDidAppear` and not `viewWillAppear` – the app is wasting ~500ms!

Changes:
- Show the spinner on any refresh not initiated by the pull-to-refresh gesture
- Move the initial refresh to `viewWillAppear`, so the screen will now show the latest content much faster